### ISSUE TITLE
Can't stop nodemon plugin on Windows

### ISSUE THREAD

**Issue by uiii** [Izhaki/nodemon-webpack-plugin#10]
Hi, I have a problem with stopping the `webpack --watch` when using nodemon plugin. When I hit Ctrl+C it will print `[nodemon] app crashed - waiting for file changes before starting...` and did't exit, still waiting for changes or `rs` command. I found the problem is with this code https://github.com/Izhaki/nodemon-webpack-plugin/blob/49c576f59652dadd7aaf63e214d0dac6b93d806e/src/index.js#L44-L49 If I remove it, it works fine then.

**Comment by Izhaki** [comment]
Aye.

This whole part is a bit more of trail and error than anything born out of understanding. Even on non-Windows platforms it is a bit of a miracle that it works.

There are two ways to go about this:
- Either you take the initiative to solve this one, researching and submitting a PR.
- Or I'll do the research, but having neither a windows machine nor a VM set-up at the moment, I'll probably submit a fix on a branch, which it would really help if you can test.

So, either way, are you happy to help?

**Comment by uiii** [comment]
I'm not sure if I have time for this. Where exactly was a problem that use that snippet? If I remove it, all is fine, at least on Windows (tried in cmd and git bash).

**Comment by Izhaki** [comment]
OK. I'll have a look tonight or tomorrow night and update.

**Comment by Izhaki** [comment]
@uiii I've just submitted a PR to fix this. Although I haven't tested it on windows, I have strong confidence it'll work.

Principally, on your fork instead of removing the snippet replace it with:
```javascript
        process.once( 'exit', () => {
            monitor.emit( 'exit' )
        })
        process.once( 'SIGINT', () => {
            process.exit( 0 )
        })
```

Would be great to learn it works.

(Also, are you sure that by removing the snippet nodemon isn't really running in the background? If you press Ctrl-c and then change the entry point, you don't see nodemon output? What if you look at the process list?)

**Comment by uiii** [comment]
Great it looks it works properly. When I hit Ctrl-C it prints `[nodemon] exiting`.

> (Also, are you sure that by removing the snippet nodemon isn't really running in the background? If you press Ctrl-c and then change the entry point, you don't see nodemon output? What if you look at the process list?)

It looks like it is running in the background and is killed when hit Ctrl-C.

**Comment by Izhaki** [comment]
This fix has now been publish to npm as version 0.1.6
